src/Microsoft.ML.HalLearners/SymSgdClassificationTrainer.cs

```diff
@@ -212,7 +212,10 @@ private TPredictor CreatePredictor(VBuffer<float> weights, float bias)
         protected override BinaryPredictionTransformer<TPredictor> MakeTransformer(TPredictor model, Schema trainSchema)
              => new BinaryPredictionTransformer<TPredictor>(Host, model, trainSchema, FeatureColumn.Name);
 
-        public BinaryPredictionTransformer<TPredictor> Train(IDataView trainData, TPredictor initialPredictor = null)
+        /// <summary>
+        /// Continues the training of a <see cref="SymSgdClassificationTrainer"/> using an initial predictor and returns a <see cref="ITransformer"/>.
+        /// </summary>
+        public BinaryPredictionTransformer<TPredictor> Fit(IDataView trainData, TPredictor initialPredictor)
             => TrainTransformer(trainData, initPredictor: initialPredictor);
```

Comment:
So many trainers have this function. Should it be moved to `TrainerEstimatorBase.cs`?

Reply:
But not all of them, right? This is not basic functionality for trainer, this is extension. We could create `TrainerWithContinuationTrainingEstimatorBase` class, and derive this classes from it, but i'm not sure it's worth it.

---
In reply to: [256977514](https://github.com/dotnet/machinelearning/pull/2528#discussion_r256977514) [](ancestors = 256977514)

Reply:
Yes unfortunately not all them have it.